library/src/window_impl.cxx

```diff
@@ -293,15 +293,6 @@ void window_impl::UpdateDynamicOptions()
   std::string hdriFile = this->Internals->Options.getAsString("render.hdri.file");
   bool hdriAmbient = this->Internals->Options.getAsBool("render.hdri.ambient");
   bool hdriSkybox = this->Internals->Options.getAsBool("render.background.skybox");
-#ifndef F3D_NO_DEPRECATED
-  std::string legacyHDRI = this->Internals->Options.getAsString("render.background.hdri");
-  if (!legacyHDRI.empty())
-  {
-    hdriFile = legacyHDRI;
-    hdriAmbient = true;
-    hdriSkybox = true;
-  }
-#endif
```

Comment:
you need this code or libf3d users will not have support for render.background.hdri option

Reply:
And the fact there is no failing test shows that I forgot to add a test for it